core/templates/dev/head/css/oppia.css

```diff
@@ -680,10 +680,17 @@ textarea {
   height: 198px;
 }
 
+.oppia-footer-margin{
+}
+
 @media (max-width: 768px) {
   .oppia-footer-padding {
     height: 464px;
   }
+
+  .oppia-footer-margin {
```

Comment:
I honestly think we can get away with just reducing the padding on the `.oppia-footer-container` on this media query for a couple reasons:

- there would be no need to add an additional class.
- this style "forces" the div to go beyond its container 60px to the left. This might become an issue in the future if the footer styles are to be changed again.

I am well aware that the code does solve the problem and renders the desired result but, for the sake of keeping our files a little bit easier to maintain (I seem to be saying that a lot these days), I think this is a better solution.

What do you think?

/cc @seanlip